From 3a4e20df2ebcd8dcf3f1695f265019618e11dcbf Mon Sep 17 00:00:00 2001
From: Jiri Benc <jbenc@redhat.com>
Date: Fri, 12 Sep 2014 14:26:53 -0400
Subject: [net] genetlink: rename shadowed variable

Message-id: <d00058d9c487b41ee13478048ee22fd92f8d2efa.1410528958.git.jbenc@redhat.com>
Patchwork-id: 93958
O-Subject: [RHEL7.1 net PATCH 11/23] genetlink: rename shadowed variable
Bugzilla: 1140656
RH-Acked-by: Jiri Pirko <jpirko@redhat.com>
RH-Acked-by: Florian Westphal <fwestpha@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1140656
Brew: https://brewweb.devel.redhat.com/taskinfo?taskID=7948291

commit 029b234fb34d53e11cadd8c1f530107f557e1aca
Author: Johannes Berg <johannes.berg@intel.com>
Date:   Mon Nov 18 20:54:58 2013 +0100

    genetlink: rename shadowed variable

    Sparse pointed out that the new flags variable I had added
    shadowed an existing one, rename the new one to avoid that,
    making the code clearer.

    Signed-off-by: Johannes Berg <johannes.berg@intel.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>

Signed-off-by: Jiri Benc <jbenc@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/net/netlink/genetlink.c b/net/netlink/genetlink.c
index 339dd72..d4b7459 100644
--- a/net/netlink/genetlink.c
+++ b/net/netlink/genetlink.c
@@ -668,21 +668,21 @@ static int ctrl_fill_info(struct genl_family *family, u32 portid, u32 seq,
   for (i = 0; i < family->n_ops; i++) {
    struct nlattr *nest;
    const struct genl_ops *ops = &family->ops[i];
-   u32 flags = ops->flags;
+   u32 op_flags = ops->flags;
 
    if (ops->dumpit)
-    flags |= GENL_CMD_CAP_DUMP;
+    op_flags |= GENL_CMD_CAP_DUMP;
    if (ops->doit)
-    flags |= GENL_CMD_CAP_DO;
+    op_flags |= GENL_CMD_CAP_DO;
    if (ops->policy)
-    flags |= GENL_CMD_CAP_HASPOL;
+    op_flags |= GENL_CMD_CAP_HASPOL;
 
    nest = nla_nest_start(skb, i + 1);
    if (nest == NULL)
     goto nla_put_failure;
 
    if (nla_put_u32(skb, CTRL_ATTR_OP_ID, ops->cmd) ||
-       nla_put_u32(skb, CTRL_ATTR_OP_FLAGS, flags))
+       nla_put_u32(skb, CTRL_ATTR_OP_FLAGS, op_flags))
     goto nla_put_failure;
 
    nla_nest_end(skb, nest);
-- 
1.7.1